modules/lvm2/udiskslinuxvolumegroup.c

```diff
@@ -586,13 +586,16 @@ handle_add_device (UDisksVolumeGroup     *_group,
 }
 
 /* ---------------------------------------------------------------------------------------------------- */
-
 static gboolean
-handle_remove_device (UDisksVolumeGroup     *_group,
+handle_remove_common (UDisksVolumeGroup     *_group,
                       GDBusMethodInvocation *invocation,
                       const gchar           *member_device_objpath,
+                      GVariant              *options,
+                      gboolean               is_remove,
                       gboolean               arg_wipe,
-                      GVariant              *options)
+                      const gchar           *authentication_error_msg,
+                      const gchar           *job_operation,
+                      UDisksThreadedJobFunc  job_func)
```

Comment:
I think the last three parameters above should be determined from ``is_remove`` locally. Otherwise somebody may pass ``is_remove=True`` and then wrong error message, job operation or job function.